posthog-node/src/feature-flags.ts

```diff
@@ -62,6 +63,7 @@ class FeatureFlagsPoller {
     this.pollingInterval = pollingInterval
     this.personalApiKey = personalApiKey
     this.featureFlags = []
+    this.featureFlagPayloads = {}
```

Comment:
quick flyby that the payloads should already be defined in `this.featureFlags` so we shouldn't need this? 🤔

Reply:
oh nvm, you're generating this 👍

Reply:
just iterating through them once upon retrieval so it doesn't need to happen everytime